### ISSUE TITLE
Simplify screenshot reference

### ISSUE THREAD

**Issue by jekuno** [ronen/rspec-page-regression#12]
I have a feature test which tests different scenarios.
For each of those tests your gem looks for the same screenshot file e.g. located at `spec/expectation/user_profile_page/expected.png`.
To point the matcher to a different file I need to provide a full image path such as
`expect(page).to match_expectation 'spec/expectation/user_profile_page/access_denied.png'`

It would be more convenient if users instead could only specify the image file name without directory and without extension such as
`expect(page).to match_expectation 'access_denied'`

**Comment by ronen** [comment]
FYI it wasn't even using the example (scenario) name in the path until #8--just released 0.4.1 with that fixed.

Since many people agree that best practice in RSpec is to have only one expectation per example, this doesn't seem like it should be encouraged.   But that said, for completeness it ought to be supported.

Happy to accept a PR!

**Comment by jekuno** [comment]
IMHO best practice is to have only one assertion per test (i.e. per scenario). Nevertheless you can have one spec file which covers a certain feature and has several scenarios per feature.  

I will see whether I find the time for making a PR.
(Sorry for opening so many issues. I really like your gem. It's just that I started using your gem on a bigger project and wanted to give feedback on possible issues ASAP).

**Comment by ronen** [comment]
> IMHO best practice is to have only one assertion per test (i.e. per scenario). Nevertheless you can have one spec file which covers a certain feature and has several scenarios per feature.

Yes, I wasn't clear.  #8 & release 0.4.1 fix that: the intended behavior is that the scenario name be included as the last component of the path, and that wasn't happening, but now it is.  

Having more than assertion per scenario would require some sort of option such as `match_reference_image label: 'my_label'`, which I think shouldn't be encouraged but should be allowed.

> I will see whether I find the time for making a PR.
> (Sorry for opening so many issues. I really like your gem. It's just that I started using your gem on a bigger project and wanted to give feedback on possible issues ASAP).

No problem at all--thanks for the ideas, they're all excellent.  I'm glad that you like the gem and would be happy to see it be made better!  If I had more time I'd happily implement all your suggestions myself.  But unfortunately too busy these days to be able to dive into it.

BTW if you do find the time to work on all the features you're suggesting, you'd end up being a pretty major contributor.  I'd be happy to give you direct access.  In fact with more than one major contributor, it'd make sense to move it out of `github.com/ronen` and create an organization such as `github.com/page-regression/` to own the repo, and likewise update the ownership on rubygems.com
